MANIFESTO.md

```diff
@@ -1 +1,6 @@
 
+##Community
+
+Mankind did not evolve by sitting alone in the dark. For the software industry to really grow in the Caribbean we need collaboration, knowledge and skill sharing. Avoid platforms and tools that do not respect communities that form around them or worse; shun them.
+
+Community does not only mean software developers, after all we need to solve somebody's problems. Developers should therefore take time to interact with and integrate with the rest of society.
```

Comment:
There are no software developers on earth who have **paying** jobs that are not about solving a problem someone else has. I say that without fear of contradiction.

Don't make assumptions though, there is no mention there of Caribbean Developers or accusations of anything. It's merely a point of view on what it takes to improve the quality of the industry locally/regionally. Some may be doing it others may not. If you disagree then explain why you don't think Community is necessary. :100: